draft-ietf-taps-interface.md

```diff
@@ -436,6 +436,14 @@ Connection.Close()
 Connection2.Close()
 ~~~
 
+Preconnections are reusable. Hence, for example, after the Connections were closed,
+the following would be correct:
```

Comment:
After Initiated wouldn't this also be OK?

Reply:
Of course - anywhere, really, it's just a struct. But this gets a bit harder to write about ("a similar line would also work below the 'initiate' call higher above"), unless we use more code just to show this; I think, with the explanation below, it should be clear enough as it is.

Reply:
Could we try: 
"Preconnections are reusable after being used to initiate a Connection. Hence, for example, after the Connections were closed,
 the following would be correct:"

Reply:
Your wish has been granted.